src/index.js

```diff
@@ -17,7 +17,7 @@ const handler = async (req, res) => {
       formattedDate: format(
         new Date(accounts[0].created * 1000),
         `MMMM Do ${new Date().getFullYear()} @ HH:mm`,
-      ),
+      ) + ' GMT',
```

Comment:
Since the format string is already a template string, you can just add this there. Additionally, you can use `Z` to represent the timezone, https://date-fns.org/v1.29.0/docs/format

Reply:
I tried adding it to the template string but it read M as a numeric month so I took it out. And because unix timestamps don't have an explicit timezone, i decided to hard code this to avoid any localization issues in converting it.

Reply:
The more I think about it, I think it's actually probably best to just remove the entire time component as well. Especially since the 'birthday' calculation only happens on the day level.

Reply:
/shrug I considered that too but you can't actually see when you signed up on your d.o profile so I thought it was kinda cool to see.

Reply:
How about doing something like, `MMMM Do ${new Date().getFullYear()} @ HH:mm Z`